source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -103,6 +104,22 @@ std::vector<std::string> initNamespaces(const Protobuf::RepeatedPtrField<std::st
   return namespaces;
 }
 
+absl::optional<envoy::config::core::v3::TypedExtensionConfig>
+getProcessingRequestModifierConfig(const ExternalProcessor& config) {
+  if (config.has_processing_request_modifier()) {
+    return config.processing_request_modifier();
+  }
+  return absl::nullopt;
+}
+
+absl::optional<envoy::config::core::v3::TypedExtensionConfig>
+initProcessingRequestModifierConfig(const ExtProcPerRoute& config) {
```

Comment:
Should this function be named something like: processingRequestModifierPerRouteConfig